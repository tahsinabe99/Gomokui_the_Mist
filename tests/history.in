who
place A1
place A3
place B3
place B1
place D4
history
resign
