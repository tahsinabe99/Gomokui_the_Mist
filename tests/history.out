B
A1A3B3B1D4
Black wins!
A1A3B3B1D4
Thank you for playing!
